### PR TITLE
New gauze customization

### PR DESCRIPTION
**Pull request by JustArt1m**
## About the PR
Add a new gauze to the customization

## Why / Balance
Just for more customization.

## Media
Screenshot 2024-09-14 132104
Screenshot 2024-09-14 132058
Screenshot 2024-09-14 132053
Screenshot 2024-09-14 132046

## Requirements
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl: Just_Art
- add: Added a head gauze to customization.

### CODE REVIEW

**Comment by UbaserB** [comment]
it looks more like hair than gauze

**Comment by IProduceWidgets** [comment]
if you keep applying it you should become a mummy.

**Review by UbaserB — COMMENTED** [review]
The sprite needs improvement, it should look more visibly like gauze/bandages as currently the form is unclear. Perhaps it just going around the forehead (like a headband) might be a better solution?

**Comment by JustArt1m** [comment]
@UbaserB What is the final solution?

**Comment by UbaserB** [comment]
> @UbaserB What is the final solution?

it still bothers me a bit.. are you on the discord?

**Comment by JustArt1m** [comment]
> > @UbaserB What is the final solution?
> 
> it still bothers me a bit.. are you on the discord?
Yes, I am on the space station 14 discord server. Or you mean my discord profile?

**Comment by UbaserB** [comment]
> Yes, I am on the space station 14 discord server. Or you mean my discord profile?

mhm just give a poke in #sprite-dungeon

**Comment by UbaserB** [comment]
have you tested that it works for other species?

**Comment by JustArt1m** [comment]
> have you tested that it works for other species?

Yes, it works on other species

**Comment by UbaserB** [comment]
> Yes, it works on other species

one last thing, can you make it so it's a bit less noisy? Right now the sprite feels like it has a ton of randomised pixels when it could have more horizontal lines of pixels

File: Resources/Locale/en-US/markings/gauze.ftl [hunk 1]
```diff
@@ -46,6 +46,9 @@ marking-GauzeUpperLegRight = Gauze Thigh Wrap (Right)
 marking-GauzeBlindfold-gauze_blindfold = Gauze Blindfold
 marking-GauzeBlindfold = Gauze Blindfold
 
+marking-GauzeHead-gauze_head = Gauze Head
+marking-GauzeHead = Gauze Head
```
Comment (UbaserB) [inline Resources/Locale/en-US/markings/gauze.ftl:50]: gauze head wrap sounds better i think

**Comment by UbaserB** [comment]
Thanks for your contribution!

**Pull request merged**